pkg/controllers/configobservation/console/observe_consoleurl.go

```diff
@@ -35,9 +35,6 @@ func ObserveConsoleURL(genericlisters configobserver.Listers, recorder events.Re
 			break
 		}
 	}
-	if !isConsoleCapabilityEnabled {
-		return existingConfig, errs
-	}
 
```

Comment:
wrap this whole piece with the condition block and remove the capability special case from the error handling